main.js

```diff
@@ -277,4 +279,5 @@ document.addEventListener('o.DOMContentLoaded', initialise);
  * interface.
  * @type {Tracking}
  */
-module.exports = tracking;
+export default tracking;
```

Comment:
I think we should only have one export here, either the default or the named.

Reply:
This is leftover from the work you already did 🙈 I wondered why both were included in all the files?

Reply:
Attempting to make it non-breaking?

Reply:
Yep, I remember doing this, I think we should not do this and should export only named.

> Attempting to make it non-breaking?

Yep!